pkg/deploy/identity-provider/deployment_keycloak.go

```diff
@@ -489,13 +490,20 @@ func getSpecKeycloakDeployment(
 		" && /opt/jboss/docker-entrypoint.sh -b 0.0.0.0 -c standalone.xml"
 	command += " -Dkeycloak.profile.feature.token_exchange=enabled -Dkeycloak.profile.feature.admin_fine_grained_authz=enabled"
 	if cheFlavor == "codeready" {
+		addUsernameReadonlyTheme := "baseTemplate=/opt/eap/themes/base/login/login-update-profile.ftl" +
+			" && readOnlyTemplateDir=/opt/eap/themes/codeready-username-readonly/login" +
+			" && readOnlyTemplate=${readOnlyTemplateDir}/login-update-profile.ftl" +
+			" && if [ ! -d ${readOnlyTemplateDir} ]; then" +
+			" mkdir -p ${readOnlyTemplateDir}" +
+			" && cp ${baseTemplate} ${readOnlyTemplate}" +
+			" && echo \"parent=rh-sso\" > ${readOnlyTemplateDir}/theme.properties" +
+			" && sed -i 's|id=\"username\" name=\"username\"|id=\"username\" readonly name=\"username\"|g' ${readOnlyTemplate}; fi"
```

Comment:
@tolusha could you please calrify when / on which stage this command is executed?
I'm wondering if it is possible to do such a manipulation manually for CRW 2.5

Reply:
The command is run inside keycloak pod before keycloak starts up.
I don't see a way how we can do it manually.